apps/user-office-backend/tsconfig.json

```diff
@@ -11,6 +11,8 @@
       "esnext.asynciterable"
     ] /* Specify library files to be included in the compilation. */,
     "allowJs": true /* Allow javascript files to be compiled. */,
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true,
```

Comment:
These don't sound like safe options to be adding.

[The docs for skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck) suggest using alternative dependency resolution tools instead of it.

[The docs for skipDefaultLibCheck](https://www.typescriptlang.org/tsconfig#skipDefaultLibCheck) say to use skipLibCheck instead, so at a minimum I don't think we need both.

Reply:
`skipLibCheck` is added for performance. They have been set for `user-office-frontend` and `user-office-frontend-e2e` tsconfigs already. It does not cause problems with development experience.

You are right the skipDefaultLibCheck should be removed.

Reply:
Done